test 2

jmutcap opened the pull request.

jmutcap committed:
test 2

jmutcap committed:
Update variables.py

added a M

jmutcap committed:
First Test Assigment

murraycoding committed:
Added data types file to explore data types

murraycoding committed:
fixed code in test.py to prevent infinite loop when running

murraycoding committed:
moved the data_types file

murraycoding committed:
added assignment 1 to the branch

murraycoding commented on Test.py:
```diff
@@ -0,0 +1,8 @@
+while count < 11:
+    print(f"number {count}")
+    if count == 7:
+        print("This is 7") # fixing so you don't have an infinite loop
```
@jmutcap example of a comment on one of your files. I changed this here to prevent you from running an infinite loop

jmutcap committed:
assigment_1 has been completed

i added a few extra coded that you did not ask for it, but check them out and let me know your feedbacks and better way of doing these.

jmutcap committed:
Update assignment_1.py

jmutcap committed:
Test excel file reader

putting some codes together to read and ask the user to select some columns to view.. just playing around

jmutcap committed:
please check the file pdftest

jmutcap committed:
PDF READER update

jmutcap committed:
DELETED A MODULE

jmutcap committed:
Automated module

jmutcap committed:
RC reader draft

jmutcap committed:
new projects draft

jmutcap committed:
soil data

jmutcap committed:
soil

jmutcap committed:
other projects
